tasks/libs/pipeline/github_slack_map.yaml

```diff
@@ -44,5 +44,5 @@
 '@datadog/agent-devx-infra': '#agent-devx-ops'
 '@datadog/agent-devx-loops': '#agent-devx-ops'
 '@datadog/apm-onboarding': '#apm-onboarding'
-'@datadog/apm-reliability-and-performance': '#apm-ecosystems-reliability'
+'@datadog/apm-ecosystems-performance': '#apm-benchmarking-platform'
```

Comment:
If there is a split, shouldn't this be two lines containing two teams?

Reply:
Thanks for question Celian!
No, the change is correct. After split, only my team owns CI code of benchmarking CI job for trace-agent.
#apm-ecosystems-reliability (the other team) doesn't own anything in datadog-agent repo as of now.

Reply:
I can't merge the changes though (see "Merging is blocked" in Github). @CelianR could you please click merge button? :pray: :bow: